src/tripCard/index.tsx

```diff
@@ -1,160 +1,2 @@
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-    height: 100%;
-  }
-
-  & .kirk-tripCard-main {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    margin-bottom: ${space.l};
-  }
-
-  & .kirk-tripCard-itinerary {
-    min-width: 0; /* reset flex default to allow shrinking below content width */
-  }
-
-  &.kirk-tripCard--with-price .kirk-tripCard-itinerary {
-    padding-right: ${space.m};
-  }
-
-  & .kirk-tripCard-price {
-    display: block;
-    flex-shrink: 0;
-    padding-top: ${space.m};
-  }
-
-  & .kirk-tripCard-bottom {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-  }
-
-  & .kirk-tripCard-bottom--only-right {
-    justify-content: flex-end;
-  }
-
-  & .kirk-tripCard-bottom-left {
-    min-width: 0; /* reset flex default to allow shrinking below content width */
-  }
-
-  & .kirk-tripCard-bottom-right {
-    flex-shrink: 0;
-  }
-
-  & .kirk-tripCard-bottom-left + .kirk-tripCard-bottom-right {
-    margin-left: ${space.l};
-  }
-
-  & .kirk-tripCard-driver,
-  & .kirk-tripCard-passengers {
-    display: flex;
-    align-items: center;
-    flex: 1;
-    min-width: 0; /* reset flex default to allow shrinking below content width */
-  }
-
-  & .kirk-tripCard-driver-info {
-    min-width: 0; /* reset flex default to allow shrinking below content width */
-  }
-
-  & .kirk-tripCard-driver-name {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    max-height: calc(2 * ${font.m.lineHeight}); /* max 2 lines of text */
-  }
-
-  & .kirk-tripCard-ratingContainer {
-    display: flex;
-    margin-top: ${space.m};
-  }
-
-  & .kirk-tripCard-rating {
-    margin-left: ${space.m};
-  }
-
-  /* We reverse the passengers array to display the 1st one on top without touching z-indexes */
-  & .kirk-tripCard-passengers {
-    flex-direction: row-reverse;
-    justify-content: flex-end;
-    list-style-type: none;
-  }
-
-  /* To match the width of the time area of the Itinerary */
-  & .kirk-tripCard-driver .kirk-tripCard-avatar {
-    flex-shrink: 0;
-    margin-right: ${space.l};
-  }
-
-  & .kirk-tripCard-passengers .kirk-tripCard-avatar {
-    margin-right: -${space.m};
-  }
-
-  & .kirk-tripCard-passengers .kirk-avatar {
-    border: 2px solid ${color.white};
-    background: ${color.white};
-  }
-
-  & .kirk-tripCard-passengers .kirk-tripCard-passengers-more {
-    background: ${color.midnightGreen};
-    border-radius: 50%;
-    width: 40px;
-    height: 40px;
-    border: 2px solid ${color.white};
-    line-height: 38px;
-    text-align: center;
-  }
-
-  & .kirk-tripCard-flags {
-    color: ${color.lightMidnightGreen};
-  }
-
-  & .kirk-tripCard-flags .kirk-icon {
-    display: inline-block;
-    margin-left: ${space.m};
-  }
-`
-
-export { TripCardProps, User } from './TripCard'
-export { StyledTripCard as TripCard }
-export default StyledTripCard
+export { TripCard, TripCardProps } from './TripCard.style'
```

Comment:
You should import `TripCard.style.tsx` only in `TripCard.tsx`. 
Look at the avatar component for an example

Reply:
Makes sense for Avatar because it's wrapped with the `StyledAvatar` but `TripCard` is not. It's wrapped with `Card` component maybe I should migrate `Card` component first.

Reply:
Actually `Card` is an internal component so there's nothing to migrate.

Reply:
I updated the `.style` so now it's overriding `internals/card` instead of `TripCard` component. PTAL @mcellier

Reply:
> > Missing changelog update
> 
> Not sure, since it's an internal change I think we said that we don't need to update the changelog. Poke @gabrielagabriel

I've seen several in the changelog already

Reply:
We have decided to update the changelog since it may break some tests on the App.

Reply:
I'll update the changelog. Thanks